program/tests/tests/limits.rs

```diff
@@ -79,7 +80,11 @@ async fn test_max_validators_maintainers() {
         let maintainer = context.add_maintainer().await;
         context.maintainer = Some(maintainer);
 
-        let validator = context.add_validator().await;
+        validator = Some(context.add_validator().await);
+    }
+
+    // take out of the loop to reduce the time to wait
```

Comment:
But now this is testing a weaker property, it only tests deposit/stake/unstake with a single validator.

Reply:
We add a new validator in the loop with `context.add_validator().await`, so plenty of validators. And at the end we just test that `deposit` and `unstake` work properly with a list of new validators.